src/reference/asciidoc/aggregator.adoc

```diff
@@ -347,214 +334,157 @@ Below you can see an example of an aggregator.
+<4> The channel to which the aggregator sends the aggregation results.
+Optional (because incoming messages can themselves specify a reply channel in the 'replyChannel' message header).
+<5> The channel to which the aggregator sends the messages that timed out (if `send-partial-result-on-expiry` is `false`).
+Optional.
 <6> A reference to a `MessageGroupStore` used to store groups of messages under their correlation key until they are complete.
-_Optional_, by default a volatile in-memory store.
-
-
-
-<7> Order of this aggregator when more than one handle is subscribed to the same DirectChannel (use for load balancing purposes).
-_Optional_.
-
-
-
-<8> Indicates that expired messages should be aggregated and sent to the 'output-channel' or 'replyChannel' once their containing `MessageGroup` is expired (see `MessageGroupStore.expireMessageGroups(long)`).
-One way of expiring `MessageGroup` s is by configuring a `MessageGroupStoreReaper`.
-However `MessageGroup` s can alternatively be expired by simply calling `MessageGroupStore.expireMessageGroups(timeout)`.
-That could be accomplished via a Control Bus operation or by simply invoking that method if you have a reference to the `MessageGroupStore` instance.
-Otherwise by itself this attribute has no behavior.
-It only serves as an indicator of what to do (discard or send to the output/reply channel) with Messages that are still in the `MessageGroup` that is about to be expired.
-_Optional_.
-_Default - false_.
-*NOTE:* This attribute is more properly `send-partial-result-on-timeout` because the group may not actually expire if
+Optional.
+By default, it is a volatile in-memory store.
+// TODO How would one change it?
+<7> The order of this aggregator when more than one handle is subscribed to the same `DirectChannel` (use for load-balancing purposes).
+Optional.
+<8> Indicates that expired messages should be aggregated and sent to the 'output-channel' or 'replyChannel' once their containing `MessageGroup` is expired (see https://docs.spring.io/spring-integration/api/org/springframework/integration/store/MessageGroupStore.html#expireMessageGroups-long[`MessageGroupStore.expireMessageGroups(long)`]).
+One way of expiring a `MessageGroup` is by configuring a `MessageGroupStoreReaper`.
+However you can alternatively expire `MessageGroup` by calling `MessageGroupStore.expireMessageGroups(timeout)`.
+You can accomplish that through a Control Bus operation or, if you have a reference to the `MessageGroupStore` instance, by invoking `expireMessageGroups(timeout)`.
+Otherwise, by itself, this attribute does nothing.
+It serves only as an indicator of whether to discard or send to the output or reply channel any messages that are still in the `MessageGroup` that is about to be expired.
+Optional (the default is `false`).
+NOTE: This attribute might more properly be called `send-partial-result-on-timeout`, because the group may not actually expire if
 `expire-groups-upon-timeout` is set to `false`.
```

Comment:
This must be join with the previous line to follow the `sentence per line` rule.